### PR TITLE
chore(extension): add extension for camel-http-common

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: extensions/http-common/runtime/pom.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        <!-- camel -->
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-http-common</artifactId>
+        </dependency>
+
+        <!-- required for http-common type converters-->
+        <dependency>
+            <groupId> org.jboss.spec.javax.servlet</groupId>
```
Comment (ppalaga) [inline extensions/http-common/runtime/pom.xml]: You might want to remove the space after `<groupId>`

File: extensions/http-common/runtime/pom.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            <artifactId>jboss-servlet-api_4.0_spec</artifactId>
+            <version>1.0.0.Final</version>
```
Comment (ppalaga) [inline extensions/http-common/runtime/pom.xml]: `jboss-servlet-api_4.0_spec` is managed in `io.quarkus:quarkus-bom`. There is no need to have the explicit version here.

**Comment by ppalaga** [comment]
What problem is this solving?

**Comment by lburgazzoli** [comment]
> What problem is this solving?

none, some components require some classes from camel-http-common thus I created the related extension

**Pull request merged**